queenbee/job/status.py

```diff
@@ -132,7 +134,12 @@ class JobStatus(BaseStatus):
 
     steps: Dict[str, StepStatus] = {}
 
+    inputs: List[StepInputs] = Field(
```

Comment:
I thought the inputs should be DAGInputs  but if we can resolve the DAG inputs to StepInputs then this is just as good.

Reply:
DAG Inputs don't contain any of the actual data used. A DAG input only has a schema and a default value. So in this case I was hesitating between `JobArguments` and `StepInputs` and decided to go for the `StepInputs` because they contain both the actual value and the schema information.

Reply:
You are correct. I meant JobArguments and I agree with your decision.